ceph.spec.in

```diff
@@ -159,7 +159,7 @@ BuildRequires:	gcc-c++
 BuildRequires:	gdbm
 %if 0%{with tcmalloc}
 %if 0%{?fedora} || 0%{?rhel}
-BuildRequires:	gperftools-devel >= 2.6.1
+BuildRequires:	gperftools-devel >= 2.8.1
```

Comment:
my only concern is that EPEL8 still does not have gperftools 2.8.1 packaged. see https://src.fedoraproject.org/rpms/gperftools/blob/epel8/f/gperftools.spec, where we have 2.7.
how about something like
```
%if 0%{?fedora}
BuildRequires:	gperftools-devel >= 2.8.1
%endif
%if %{?rhel}
BuildRequires:	gperftools-devel >= 2.7
%endif
```
?

Reply:
Yeah we need to update EPEL 8 first here.

Reply:
To update gperftools in epel8, you can write a pull request against the epel8 branch here: https://src.fedoraproject.org/rpms/gperftools/tree/epel8 . It would be a good idea to choose the Git commit from the Rawhide branch that you need, and then simply `git merge` that into `epel8`

Reply:
@ktdreyer, @tchaikov  : I believe the gperftools bump has landed.  Considering these comments as resolved.

Reply:
no it's only in testing atm

Reply:
and it's 2.8, not 2.8.1

Reply:
It has not landed. Please see https://bodhi.fedoraproject.org/updates/FEDORA-EPEL-2021-dd6932436d . Would you please test that build and provide feedback in Bodhi? Bodhi controls when that build goes to EPEL's stable repo.

Reply:
i concur with @yselkowitz on both comments. FWIW, the package is still in its testing stage: https://bodhi.fedoraproject.org/updates/FEDORA-EPEL-2021-dd6932436d 

so i am unresolving this thread.

Reply:
The other problem with this is that the same spec file is used to build for both RHEL 7 and 8, but we're only looking at 8.  This would break the RHEL 7 build since we're not bumping gperftools there.

Reply:
@yselkowitz i think it's fine as we don't build the latest stable release (pacific) or master on RHEL7 anymore. for instance, see 
- pacific builds: 
https://shaman.ceph.com/builds/ceph/pacific/e6acac2d95e64b6e78e2331a68680d107b8706bf/
- master builds: https://shaman.ceph.com/builds/ceph/master/8d7cf601a2c45d5b29c76ade42bd726e46ace161/

we are still building octopus on RHEL7/CentOS7 though. see

- https://shaman.ceph.com/builds/ceph/octopus/9b5e7e2f397c58f950a93c45ea92755d259c2fca/